common/service/dynamicconfig/config_test.go

```diff
@@ -229,11 +227,10 @@ func (s *configSuite) TestGetDurationPropertyFilteredByTaskListInfo() {
 	key := testGetDurationPropertyFilteredByTaskListInfoKey
 	domain := "testDomain"
 	taskList := "testTaskList"
-	taskType := 0
 	value := s.cln.GetDurationPropertyFilteredByTaskListInfo(key, time.Second)
-	s.Equal(time.Second, value(domain, taskList, taskType))
+	s.Equal(time.Second, value(domain, taskList, 0))
```

Comment:
Interestingly, literals will autocast to sized typed (here its being interpreted as int32 now)